Source/CombatExtended/CombatExtended/SuppressionUtility.cs

```diff
@@ -23,7 +24,7 @@ public static class SuppressionUtility
 
         private static DangerTracker dangerTracker;
 
-        private static List<CompProjectileInterceptor> interceptors;
+        private static IEnumerable<CompProjectileInterceptor> Interceptors(Thing pawn) => pawn.Map.listerThings.ThingsInGroup(ThingRequestGroup.ProjectileInterceptor).Select(t => t.TryGetComp<CompProjectileInterceptor>());
```

Comment:
This probably needs to be cached per tick and map, since iterating all things, even by group, is on the expensive side.